wolfcrypt/src/asn.c

```diff
@@ -2262,6 +2266,20 @@ const byte* OidFromId(word32 id, word32 type, word32* oidSz)
     return oid;
 }
 
+#ifdef WOLFSSL_OID_IPSEC_IKE
+static const byte* OidFromId_conflict(const byte *checkOid, const byte *actualOid)
```

Comment:
Will this method allow for growth, if we find other extension OID values that conflict?

Reply:
In some extent, yes.
It follows steps below, though way to check may differ if you avoid exact match by memcmp.

check if it is conflicted counter OID
    then return counter
    else return the same